### PR TITLE
Remove Cookie Consent UI from all templates and move into docs #10346

### PR DESCRIPTION
**Pull request by isaacrlevin**
Summary of the changes (Less than 80 chars)
 - Remove Cookie Consent Partials from Templates
 - Remove Cookie Consent Middleware from Templates

Addresses #10346

### CODE REVIEW

File: src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-FSharp/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -39,37 +30,16 @@
             </div>
         </nav>
     </header>
-    <div class="container">
-        <partial name="_CookieConsentPartial" />
-        <main role="main" class="pb-3">
-            @RenderBody()
-        </main>
```
Comment (brockallen) [inline src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-FSharp/Views/Shared/_Layout.cshtml]: Does this mean the page body won't render anymore?

File: src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -44,39 +34,15 @@
             </div>
         </nav>
     </header>
-    <div class="container">
-        <partial name="_CookieConsentPartial" />
-        <main role="main" class="pb-3">
-            @RenderBody()
-        </main>
```
Comment (brockallen) [inline src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Views/Shared/_Layout.cshtml]: Does this mean the page body won't render anymore?

Comment (isaac2004) [reply src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Views/Shared/_Layout.cshtml]: Ugh yea you are right, thanks for the quick catch. I fixed this in the 3 layout views

**Comment by ryanbrandenburg** [comment]
@isaac2004 looks good, we'll have to keep #10346 open to track doing the doc work.

**Pull request merged**